e2e/process-services-cloud/process-custom-filters.e2e.ts

```diff
@@ -28,74 +28,182 @@ import { AppListCloudPage, LocalStorageUtil } from '@alfresco/adf-testing';
 import resources = require('../util/resources');
 
 import { browser, protractor } from 'protractor';
+import CONSTANTS = require('../util/constants');
 
 describe('Process list cloud', () => {
 
-    describe('Process List', () => {
-        const settingsPage = new SettingsPage();
-        const loginSSOPage = new LoginSSOPage();
-        const navigationBarPage = new NavigationBarPage();
-        const appListCloudComponent = new AppListCloudPage();
-        const processCloudDemoPage = new ProcessCloudDemoPage();
-        const tasksCloudDemoPage = new TasksCloudDemoPage();
-
-        let tasksService: TasksService;
-        let processDefinitionService: ProcessDefinitionsService;
-        let processInstancesService: ProcessInstancesService;
-        let queryService: QueryService;
-
-        let completedProcess, runningProcessInstance, switchProcessInstance, noOfApps;
-        const candidateuserapp = resources.ACTIVITI7_APPS.CANDIDATE_USER_APP.name;
-
-        beforeAll(async (done) => {
-            settingsPage.setProviderBpmSso(TestConfig.adf.hostBPM, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, false);
-            loginSSOPage.clickOnSSOButton();
-            loginSSOPage.loginSSOIdentityService(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+    const settingsPage = new SettingsPage();
+    const loginSSOPage = new LoginSSOPage();
+    const navigationBarPage = new NavigationBarPage();
+    const appListCloudComponent = new AppListCloudPage();
+    const processCloudDemoPage = new ProcessCloudDemoPage();
+    const tasksCloudDemoPage = new TasksCloudDemoPage();
+
+    let tasksService: TasksService;
+    let processDefinitionService: ProcessDefinitionsService;
+    let processInstancesService: ProcessInstancesService;
+    let queryService: QueryService;
+    let activitiUser, activitiUserRoleId;
+
+    let completedProcess, runningProcessInstance, switchProcessInstance, noOfApps;
+    const candidateuserapp = resources.ACTIVITI7_APPS.CANDIDATE_USER_APP.name;
+
+    const apiService = new ApiService('activiti', TestConfig.adf.hostBPM, TestConfig.adf.hostSso, 'BPM');
+    const identityService = new IdentityService(apiService);
+    const rolesService = new RolesService(apiService);
+    beforeAll(async (done) => {
+        settingsPage.setProviderBpmSso(TestConfig.adf.hostBPM, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, false);
+        loginSSOPage.clickOnSSOButton();
+        loginSSOPage.loginSSOIdentityService(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        await LocalStorageUtil.setConfigField('adf-edit-process-filter', JSON.stringify({
```

Comment:
can EditProcessFilterConfiguration class be used here, instead?

Reply:
Can do it that way

Reply:
actually, we are not using the class, using => await LocalStorageUtil.setConfigField
is preferable, as that way it's quicker to run the test.